### PR TITLE
Fix empty node group

### PR DESCRIPTION
**Pull request by itkg-alavieille**

### CODE REVIEW

File: ApiBundle/Controller/NodeController.php [hunk 1]
```diff
@@ -168,9 +168,14 @@ public function listNodeByAuthorAndSiteIdAction($published)
     public function listTreeNode($siteId)
     {
         $nodes = $this->get('open_orchestra_model.repository.node')->findLastVersionByType($siteId);
-        $orderedNodes = $this->get('open_orchestra_display.manager.tree')->generateTree($nodes);
 
-        return $this->get('open_orchestra_api.transformer_manager')->get('node_tree')->transform(end($orderedNodes));
+        if (!empty($nodes)) {
```
Comment (nicolasThal) [inline ApiBundle/Controller/NodeController.php]: you could invert the test by writing if (empty($nodes)) { then return array(); }

**Comment by itkg-alavieille** [comment]
@nicolasThal

File: ApiBundle/Controller/NodeController.php [hunk 2]
```diff
@@ -168,9 +168,14 @@ public function listNodeByAuthorAndSiteIdAction($published)
...
     {
         $nodes = $this->get('open_orchestra_model.repository.node')->findLastVersionByType($siteId);
+        if (empty($nodes)) {
+            return array();
+        }
+
         $orderedNodes = $this->get('open_orchestra_display.manager.tree')->generateTree($nodes);
 
         return $this->get('open_orchestra_api.transformer_manager')->get('node_tree')->transform(end($orderedNodes));
+
```
Comment (nicolasThal) [inline ApiBundle/Controller/NodeController.php]: you could also suppress this empty line

**Comment by nicolasThal** [comment]
:+1: with the line suppressed

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**